builtin/providers/circonus/resource_circonus_check_http_test.go

```diff
@@ -0,0 +1,180 @@
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.1136493216.unit", "seconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.active", "true"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.name", "tt_connect"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.type", "numeric"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.4246441943.unit", "milliseconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.active", "true"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.name", "tt_firstbyte"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.type", "numeric"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "stream.3695203246.unit", "milliseconds"),
+
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.#", "4"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.30226350", "app:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.3219687752", "app:jezebel"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.1543130091", "lifecycle:unittests"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "tags.3241999189", "source:circonus"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "target", "127.0.0.1"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "type", "http"),
+				),
+			},
+		},
+	})
+}
+
+const testAccCirconusCheckHTTPConfig = `
+variable "http_check_tags" {
+  type = "list"
+  default = [ "app:circonus", "app:jezebel", "lifecycle:unittests", "source:circonus" ]
+}
+
+resource "circonus_metric" "status_code" {
+  name = "code"
```

Comment:
I took the preventative step of fully randomizing all resource names.  I do not believe this was necessary but just in case we make a radical change to our test runners in the future and go fully parallel on all tests, we're now covered.